troposphere/static/js/actions/AccountActions.js

```diff
@@ -37,8 +37,20 @@ export default {
             }
 
         }).fail(function(response) {
-            var err_response = response.responseJSON || response.responseText;
-            var message = "Error creating Account " + account.get("atmo_user") + ":" + err_response;
+            var error_obj = response.responseJSON;
+            var error_msg = response.responseText;
+            if(error_obj) {
+                error_msg = "";
+                let error_type;
+                for (error_type in error_obj) {
+                    let value = error_obj[error_type];
+                    if(Array && Array.isArray(value)) {
+                        value = value[0];
+                    }
+                    error_msg += error_type + ":" + value
+                }
+            }
```

Comment:
we might consider this a "Actions" only utility since it has to do with the response handling within _actions_. I'd be cool with `../actions/utilities/errorHandler.js` (maybe?)